tutorial/template-tutorial.md

```diff
@@ -1,45 +1,87 @@
-Make sure you meet the following prerequisites before starting the tutorial steps:
+## Overview
 
-* Prerequisite one
-* Prerequisite two
-* etc.
+In this tutorial, you'll learn how to {learning objective} using {product}. This tutorial is intended for {audience}. It assumes that you have basic knowledge of {concept 1}, {concept 2}, and {concept 3}. 
 
-# Part 1 - <_Part Title_>
+## Background 
 
-{Introduce what your audience will learn in this step, then continue to write the steps in the tutorial.
-You can choose one of these approaches to write your tutorial part:
+{Use this section to provide background information about the task users will complete. For example, if your tutorial is about a particular feature, you can describe it here. If your tutorial is project-based, you can describe the project users will complete. See the ["About the Background section"](about-tutorial.md#about-the-background-section) in the template guide for more information.}
 
-* In a narrative style if your parts are short or you are using screenshots to do most of the talking.   
-* In a "Goal > Steps > Outcome" structure to build a predictable flow in all your tutorial parts.
+## Before you begin 
 
-Whatever option you choose when designing your tutorial should be carried through in subsequent parts.
-}
+{Use this section to tell users about any steps they need to complete before they start the tutorial.}
 
-# Part 2 - <_Part Title_>
+Before you start the tutorial, complete the following steps:
 
-{
-Continue the design approach you chose in the previous part and continue it through to the end of the tutorial.
-}
+* Prerequisite 1
+* Prerequisite 2
+* Prerequisite 3
 
-# What you've learned
+## Part 1 - {Description}
 
-{Summarise what knowledge the reader has gained by completing the tutorial, including a summary of each part's Goals (this is a good way to validate whether your tutorial has covered all you need it to.)}
+Feel free to use this section to summarize what users will complete in the following steps.
+
+### Step 1 - {Task name}
+
+{Provide background information for the task here. You might want to define any terminology here and provide any overview of this task's purpose.
+Include any code samples, snippets, or screenshots that will help your user complete the task}.
+
+{If users should see a result after completing this step, make sure to include it.}
+
+#### Substep 1 - {Subtask name}
+
+{Beware of asking your users to do too much in one step. If needed, break a step into one or two smaller substeps.}
+
+### Step 2 - {Task name}
+
+{Provide background information for the task here. You might want to define any terminology here and provide any overview of this task's purpose.
+Include any code samples, snippets, or screenshots that will help your user complete the task}.
+
+{If users should see a result after completing this step, make sure to include it.}
```

Comment:
**Suggestion (non-blocking):** Can you provide some guidance in your about-tutorial document about how to present results or explain more about why it helps tutorial users to see the expected result? In essence, it helps them to see if they did the tutorial correctly.